src/Validation/Validation.php

```diff
@@ -260,41 +316,53 @@ public static function comparison($check1, $operator, $check2)
         $operator = str_replace([' ', "\t", "\n", "\r", "\0", "\x0B"], '', strtolower($operator));
         switch ($operator) {
             case 'isgreater':
```

Comment:
Do we still need these strange magic strings then? I would remove those in favor of speaking const names now :)

Reply:
Those should probably stay for BC.

Reply:
`equalto` vs `notequal` are not even following the same naming pattern.. :)
I would expect the first to be `equal` only.

Reply:
It's probably out of the scope of this PR for now :)

Reply:
We could emit deprecation warnings when the string words are used. That would enable us to remove them in 4.0.0.

Reply:
@markstory deprecation warnings added